project/lib/screens/home_screen.dart

```diff
@@ -7,12 +7,13 @@ import 'package:project/styles/theme.dart';
 /// Screen/[Scaffold] the user first sees when logging on or finishing signing up.
 /// Displays a bottom tab bar for navigating to other screens.
 class HomeScreen extends StatefulWidget {
+  const HomeScreen({super.key});
+
   /// Named route for this screen.
   static const routeName = "/index";
 
   /// Creates an instance of home screen. A [Scaffold] displaying a
   /// bottom tab bar for other screens.
```

Comment:
You changes position of the constructor, you should move the documentation for it with it 😊

Reply:
Is fixed in latest commit 👍